src/models/FormationSerices.js

```diff
@@ -0,0 +1,57 @@
+import { createRequire } from "module";
+const require = createRequire(import.meta.url);
+const EntitySchema = require("typeorm").EntitySchema;
+
+export const FormationServicesSchema = new EntitySchema({
+  name: "FormationServices",
+  tableName: "formation_services",
+  columns: {
+    id: {
+      primary: true,
+      type: "int",
+      generated: "increment",
+    },
+    name: {
+      type: "varchar",
+    },
+    type: {
+      type: "varchar",
+    },
```

Comment:
Suggestion: use type as an enum with the defined values (course, workshop, etc.) to ensure the consistency from the database

Reply:
Done